entrypoint.d/45-geoip

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+if [[ "$GEOIP_ACCOUNT_ID" != "" && "$GEOIP_LICENSE_KEY" != "" ]] ; then
```

Comment:
I'm not sure but I think this won't do what you expect; try:

```suggestion
if [ -n "$GEOIP_ACCOUNT_ID" -a -n "$GEOIP_LICENSE_KEY" ] ; then
```

Docs: http://www.tldp.org/LDP/Bash-Beginners-Guide/html/sect_07_01.html